test/test_docker/test_reverse_proxy_root_path/test_reverse_proxy_root_path.py

```diff
@@ -15,6 +15,7 @@ def launch_services(launch_services_fn):
     yield from launch_services_fn(TEST_NAME, folder, "/gradio/demo", "/gradio/demo")
 
 
+@pytest.mark.serial
```

Comment:
What was the error when trying to run them in parallel?

Reply:
The Docker tests would all time out when trying to make the network request, I'm not sure why (maybe there's a shared resource that they need)